sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -1026,13 +1026,79 @@ class AdaptiveQueryExecSuite
     Seq(true, false).foreach { enableAQE =>
```

Comment:
we can merge this test cases to your two newly added test cases.

Reply:
i.e. one test to test `repartition`, and it verifies both the initial partition number and the coalesced partition number. The other test tests the same thing but for `repartitionByRange`.

Reply:
Yeah, merged them.